src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -11,32 +11,53 @@ redirects:
   - /docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-new-relic-ai
   - /docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence
 ---
+import anomalydashboard from 'images/anomaly-dashboard.png'
 
-With applied intelligence's anomaly detection, anomalies from your APM-monitored applications are automatically surfaced in our activity streams and anomalies feed. You can click each anomaly to bring up an automatic analysis.
+import anomalydetails from 'images/anomaly-details.png'
 
-Notifications for anomalies can be delivered in Slack, or you can set up a webhook to deliver messages when you need them. These events are available for querying, creating custom dashboards, and alerting. After you set up a anomaly detection configuration (a group of apps you're interested in), you can add this configuration as a source. Then the anomalies will be automatically correlated with other data sources via incident intelligence.
+import anomalyslacknotifications from 'images/anomaly-slack-notifications.png'
 
-To learn where anomalies appear, how you can use them to understand potential issues before they become incidents, and create alerts from them, watch this short video (approx. 4:15 minutes).
+With applied intelligence's anomaly detection, New Relic alerts your team of any unusual behavior. New Relic observes your APM-monitored applications and understands baseline, or normal, performance. If behavior deviates from the baseline, we know right away and alert your team so you can address any errors promptly and efficiently. 
 
-<Video
-  id="Dhuo2GdR_7o"
-  type="youtube"
+<img
+  title="A screenshot of the anomaly dashboard in the New Relic UI"
+  alt="A screenshot of the anomaly dashboard in the New Relic UI"
+  src={anomalydashboard}
 />
+<figcaption>The anomaly dashboard where your team can monitor any unusual behavior in your system.   
+  </figcaption>
 
-## Requirements
+## How we use anomalies [#anomalies-in-use]
 
-To use anomaly detection, ensure you have:
+At New Relic, we have our own developers who monitor the health of our own application. Our developers want to know if there are any outliers in our system's performance as soon as possible because we want to make sure that our customers are always able to access the data they need. New Relic's anomaly detection monitors three key golden signals: throughput, error rate and latency. Since we monitor our own application, New Relic developers always understand our baseline performance for these metrics. 
```

Comment:
```suggestion
At New Relic, our own developers monitor the health of our application. They want to know if there are any outliers in our system's performance as soon as possible because our customers should always have access to the data they need. New Relic's anomaly detection monitors three key golden signals: throughput, error rate and latency. Since we monitor our own application, New Relic developers always understand our baseline performance for these metrics. 
```

I shortened this excerpt to be shorter so it looks more scannable on the page. Didn't alter the actual writing, but just shifted it a bit. This is great framing btw. My only suggestion is to tighten the last sentence here, maybe? Something like "Since we monitor our own application, our developers use these metrics to XYZ" idk something like that to make it a tighter paragraph